sdcm/cluster.py

```diff
@@ -3498,6 +3513,7 @@ def __init__(self, *args, **kwargs):
         self.nemesis_threads = []
         self.nemesis_count = 0
         self.test_config = TestConfig()
+        self.node_benchmark_manager = ScyllaClusterBenchmarkManager()
```

Comment:
why do you need same on cluster level? may be rename to self.cluster_benchmark_manager?

Reply:
or you can init it here (even it as singleton) and in basenode get reference from self.parent_cluster.self.node_benchmark_manager

Reply:
Good catch. This looks to be redundant - I removed it and left the `ScyllaClusterBenchmarkManager` only in the `BaseCluster` class.